modules/logging/filters.go

```diff
@@ -185,15 +187,120 @@ func (m IPMaskFilter) Filter(in zapcore.Field) zapcore.Field {
 	return in
 }
 
+type ActionType string
```

Comment:
Does this need to be a separate type? I think we could probably just use string, since this type doesn't have any special properties or behaviors. (And I suspect it definitely doesn't need to be exported.)

Reply:
The idea was to add `IsValid()` on this, see my review above https://github.com/caddyserver/caddy/pull/4424#discussion_r753632173